themes/src/main/resources/theme/base/login/resources/js/authChecker.js

```diff
@@ -1,6 +1,15 @@
 const CHECK_INTERVAL_MILLISECS = 2000;
 const initialSession = getSession();
 
+let timeout;
+
+addEventListener("beforeunload", () => {
```

Comment:
Let's add a comment here describing why we need this code, so that future devs be able to understand it at a glance.

Reply:
:+1:  Done!